migrations/20231117164230_add_id_pkey_identities.up.sql

```diff
@@ -0,0 +1,29 @@
+do $$
+begin
+    if not exists(select * 
+        from information_schema.columns
+        where table_schema = '{{ index .Options "Namespace" }}' and table_name='identities' and column_name='provider_id')
+    then
+        alter table if exists {{ index .Options "Namespace" }}.identities 
+        rename column id to provider_id;
+    end if;
+end$$;
+
+alter table if exists {{ index .Options "Namespace" }}.identities 
```

Comment:
I guess these changes are from the previous PR?

Reply:
yup, i'll merge the other PR in before this one